components/results_table/commons/results_table_base.lua

```diff
@@ -89,6 +92,10 @@ function BaseResultsTable:readConfig()
 		(config.queryType == SOLO_TYPE and (tonumber(args.playerLimit) or DEFAULT_VALUES.playerLimit))
 		or tonumber(args.coachLimit) or DEFAULT_VALUES.coachLimit
 
+	if config.queryType == TEAM_TYPE and Table.isNotEmpty(config.aliases) then
+		config.isNotAlias = BaseResultsTable._getOpponentTemplates(config.opponent)
```

Comment:
isNotAlias is a table? With `is` I would expect a boolean.

Reply:
we can rename it
it is a list of the tts of the main entry, i.e. the non aliases

how about `config.nonAliasTeamTemplates`?